### PR TITLE
[web] Update to PatternFly v5

### PR DESCRIPTION
**Pull request by dgdavid**
## Problem

[PatternFly React](https://github.com/patternfly/patternfly-react) has released its 5th version back in [27th July 2023](https://github.com/patternfly/patternfly-react/releases/tag/v5.0.0). It comes with a lot of improvements and a bunch of breaking changes.

Agama it's young enough to be stuck in [PatternFly v4](http://v4-archive.patternfly.org/v4/).

## Solution

Move to PatternFly 5 by following the [upgrade guide](https://www.patternfly.org/get-started/upgrade) and checking that everything work and look as expected.

## Testing

- Added a new unit test, if needed
- Adapted needed unit tests
- Tested manually

## Note for reviewers

Please, _deploy_ the branch locally and test the UI from the end user perspective as part of the review. Thank you

### CODE REVIEW

File: web/src/components/core/PageOptions.jsx [hunk 1]
```diff
@@ -57,55 +60,76 @@ const Group = ({ children, ...props }) => {
...
+ * "teleported" to the header, close to the action for opening the Sidebar
  *
  * @example <caption>Usage example</caption>
  *   <PageOptions>
- *     <PageOptions.Item
- *       key="reprobe-link"
- *       description="Run a storage device detection"
- *     >
+ *     <PageOptions.Options>
+ *       <PageOptions.Option
```
Comment (imobachgs) [inline web/src/components/core/PageOptions.jsx:108]: 👍🏻

**Review by imobachgs — APPROVED** [review]
Thanks! I have tried and it seems to work quite well.

**Comment by joseivanlopez** [comment]
NP: the styles of some buttons is gone (Actions, Installation device and Custom devices):

Screenshot from 2023-10-02 23-12-25

Screenshot from 2023-10-02 23-09-20

**Comment by dgdavid** [comment]
> NP: the styles of some buttons is gone (Actions, Installation device and Custom devices):
> 
> Screenshot from 2023-10-02 23-12-25
> 
> Screenshot from 2023-10-02 23-09-20

Good catch, I'll fix it.

**Review by joseivanlopez — APPROVED** [review]

**Pull request merged**